templates/1_set_up_toffy.ipynb

```diff
@@ -15,9 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            normalize.write_mph_per_mass(base_dir=fov_path, output_dir=fov_path, fov='fov-1-scan-1', masses=moly_masses)
</pre><p>Would be aesthetically better to reduce the tab size to 4 spaces. Same for the line below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>